commands/management/add_mysql.go

```diff
@@ -176,6 +180,35 @@ func (cmd *addMySQLCommand) Run() (commands.Result, error) {
 		}
 	}
 
+	if cmd.DefaultsFile != "" {
```

Comment:
let's extract this block to separate method and write test for it.

Reply:
I wrote test which proof that username and password stay same if you dont specify defaults file, but I dont know if it is enough. Othervise I can write test, which find out working directory during test, create config file there, test it and delete it.